lib/dialects/sqlite/query-generator.js

```diff
@@ -332,7 +332,7 @@ const QueryGenerator = {
 
   quoteIdentifier(identifier) {
     if (identifier === '*') return identifier;
-    return Utils.addTicks(identifier, '`');
+    return Utils.addTicks(Utils.removeTicks(identifier, '`'), '`');
```

Comment:
This is probably one of the most important changes features that prevents injection.  Similar to how values are escaped, this escapes identifiers by removing them.  See line 580 of `postgres/query-generator.js`.